java-frontend/src/main/java/org/sonar/java/se/MethodBehavior.java

```diff
@@ -55,8 +55,11 @@ public void createYield(ProgramState programState) {
 
     if (!isConstructor() && !isVoidMethod()) {
       SymbolicValue resultSV = programState.peekValue();
-      // FIXME Handle exception path
-      if (resultSV != null) {
+      if (resultSV == null) {
+        // FIXME Handle exception path: for now ignore completely output through exceptions
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1134)